app/views/CartPage/index.tsx

```diff
@@ -0,0 +1,201 @@
+                            <NumberInput
+                                name="quantity"
+                                value={quantity}
+                                onChange={undefined}
+                            />
+                        </div>
+                    </div>
+                </Container>
+                <div className={styles.wishListButton}>
+                    <Button
+                        name={undefined}
+                        onClick={undefined}
+                        variant="secondary"
+                        icons={<FaHeart />}
+                        autoFocus
+                    >
+                        Add to wish list
+                    </Button>
+                    <Button
+                        name={undefined}
+                        variant="secondary"
+                        icons={<AiTwotoneDelete />}
+                        autoFocus
+                        onClick={deleteCartItem}
+                    >
+                        Remove
+                    </Button>
+                </div>
+            </div>
+        </div>
+    );
+}
+
+function CartPage() {
+    const [page, setPage] = useState<number>(1);
+    const [pageSize, setPageSize] = useState<number>(10);
+    const [email, setEmail] = useState<string>('admin@gmail.com');
+    const [deleteCartItem] = useMutation<
+        DeleteCartItemsMutation,
+        DeleteCartItemsMutationVariables
+    >(DELETE_CART_ITEMS);
+
+    const { data, refetch, loading } = useQuery<
+        CartListQuery,
+        CartListQueryVariables
+    >(CART_LIST, {
+        variables: { page, pageSize, email },
+    });
+
+    const removeCartItem = (id: string) => {
+        deleteCartItem({ variables: { id } });
+        refetch();
+    };
+    return (
+        <>
+            <div className={styles.wishList}>
+                {!loading && data?.cartItems?.results
+                    && (
+                        <>
+                            {
+                                data.cartItems.results.map((b: any) => (
+                                    <CartContent
+                                        id={b.book.id}
+                                        title={b.book.title}
+                                        image={b.book.image}
+                                        price={b.book.price}
+                                        quantity={b.quantity}
+                                        authors={b.book.authors}
+                                        removeBookFromCartList={removeCartItem}
+                                        cartId={b.id}
+                                    />
+                                ))
+                            }
+                        </>
+                    )}
+            </div>
+            <Button
+                name={undefined}
+                variant="secondary"
+                autoFocus
+            >
+                Order
+            </Button>
```

Comment:
Let's handle this or disable this button and add a FIXME/TODO comment

Also, why add "autoFocus" on button?